notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Lets add two notes here: </p><p>1- this is needed if you are a developer of the model and you have already changed some of the MATLAB scripts. </p><p>2- make sure that you are on the right branch of STEMMUS_SCOPE repository.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Notes added.</p>